utils/metrics.py

```diff
@@ -0,0 +1,19 @@
+from prometheus_client import Gauge
+from prometheus_client import Counter
+from prometheus_client import Histogram
+
+
+run_time = Gauge(name='qontract_reconcile_last_run_seconds',
+                 documentation='Last run duration in seconds',
+                 labelnames=['integration', 'shards', 'shard_id'])
+
+run_status = Counter(name='qontract_reconcile_run_status',
+                     documentation='Status of the runs',
+                     labelnames=['integration', 'status',
+                                 'shards', 'shard_id'])
+
+func_runtime = Histogram(name='qontract_reconcile_function_runtime',
+                         documentation='Run time for tracked functions',
+                         labelnames=['name'],
+                         buckets=(.1, .5, 1.0, 5.0, 10.0, 50.0, 100.0, 500.0,
```

Comment:
What do you thing about this?
```suggestion
                         buckets=(10.0, 30.0, 60.0, 150.0, 300.0, 600.0, 1200.0, 1800.0,
```

Reply:
sure +1